modules/host/update.go

```diff
@@ -113,8 +113,8 @@ func (h *Host) ProcessConsensusChange(cc modules.ConsensusChange) {
 			for i, txn := range block.Transactions {
 				// Check for file contracts.
 				if len(txn.FileContracts) > 0 {
-					for _ = range txn.FileContracts {
-						fcid := txn.FileContractID(uint64(i))
+					for j = range txn.FileContracts {
```

Comment:
it looks like you don't use the `i` in the `block.Transactions` range above, so you could use `i` instead. That's what you do in the `AppliedBlocks` loop.